datafusion/core/src/physical_plan/aggregates/row_hash.rs

```diff
@@ -227,11 +227,6 @@ fn group_aggregate_batch(
     for group_values in grouping_by_values {
         let group_rows: Vec<Vec<u8>> = create_group_rows(group_values, group_schema);
 
-        // evaluate the aggregation expressions.
```

Comment:
I've noticed that in the master, aggregation expression is continuously evaluated for each loop which is not same as `GroupedHashAggregateStream`

Reply:
@Dandandan  you might be interested in this part